compareECLFiles.cmake

```diff
@@ -1007,6 +1014,13 @@ if(MPI_FOUND)
                                        REL_TOL ${rel_tol_parallel}
                                        TEST_ARGS --linear-solver-reduction=1e-7 --tolerance-cnv=5e-6 --tolerance-mb=1e-8)
 
+  add_test_compare_parallel_simulation(CASENAME spe1_gaswater
+                                       FILENAME SPE1CASE2_GASWATER
+                                       SIMULATOR flow
+                                       ABS_TOL ${abs_tol_parallel}
+                                       REL_TOL ${rel_tol_parallel}
+                                       TEST_ARGS --linear-solver-reduction=1e-7 --tolerance-cnv=5e-6 --tolerance-mb=1e-8)
```

Comment:
this needs DIR

Reply:
Added DIR. Why do certain cases require the specification of DIR and others not?

Reply:
the default is that DIR is the same as the test name.